test-alerts.cpp

```diff
@@ -6,3 +6,54 @@
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
 }
+
+TEST_CASE("infers the breach according which is too high") {
+  REQUIRE(inferBreach(32, 20, 30) == TOO_HIGH);
+}
+
+TEST_CASE("infers the breach according which is too normal") {
+  REQUIRE(inferBreach(30, 20, 30) == NORMAL);
+}
+
+TEST_CASE("Infers If Email Sending Working -CASE1") {
+  BatteryCharacter batterychar = {HI_ACTIVE_COOLING, "NORMAL"};
+  REQUIRE(checkAndAlert(TO_EMAIL, batterychar,30 ) == PASS);
```

Comment:
This `REQUIRE` states that checkAndAlert converts the parameters to a PASS return value.
If you had to re-state the functionality of checkAndAlert in terms of data-conversion, what would that be?